packages/pwa-kit-react-sdk/scripts/upload.js

```diff
@@ -63,7 +63,7 @@ const uploadBundle = (customOptions) => {
         Utils.fail('[Error: You must provide a Runtime Admin project slug to upload a bundle.]')
     }
 
-    const options = Object.assign(OPTION_DEFAULTS, customOptions)
+    const options = Object.assign({}, OPTION_DEFAULTS, customOptions)
```

Comment:
I've also searched the entire monorepo for this kind of `Object.assign` bug, and this is the only place.